### PR TITLE
Changed subject to an object, added some ascii art

### PR DESCRIPTION
**Pull request by floridop**
Changed content of your story. You are free to ask me to do any change you like before approving!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+|  \ ||\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\| |     --Keith Wright
+|  |\  ,--------------------------------  |
+|  ||| |                               || |
+ \ ||| |           -  -                || |
+  \'|| |-----------\\-\\---------------|| |
+    \|_|            "  "               \|_|
+
+------------------------------------------------
+Thank you for visiting https://asciiart.website/
+This ASCII pic can be found at
```
Comment (AsharKamal98) [inline README.md:25]: Please add a semi-colon at the end of this line.

Comment (floridop) [reply README.md:25]: Maybe you mean a colon?

Comment (AsharKamal98) [reply README.md:25]: Oh yes, my bad!

**Review by AsharKamal98 — APPROVED** [review]
Looks good, accepting pull request now

**Comment by floridop** [comment]
I'll close this part of the homework, but if there is something else 
you're curious to try and I am available I'll be happy to test.

**Comment by AsharKamal98** [comment]
I think I've got the hang of it, thanks!

**Pull request merged**